include/pcg_random.hpp

```diff
@@ -561,16 +561,18 @@ operator<<(std::basic_ostream<CharT,Traits>& out,
     auto space = out.widen(' ');
     auto orig_fill = out.fill();
 
-    out << rng.multiplier() << space
-        << rng.increment() << space
-        << rng.state_;
+    pcg_extras::Output(out, rng.multiplier());
+    out << space;
+    pcg_extras::Output(out, rng.increment());
+    out << space;
+    pcg_extras::Output(out, rng.state_);
+    out << space;
```

Comment:
Note this change is intentional: the current parsing routine for pcg128_t is actually *broken* at EOF.  (This only showed up with this change, as it's not used anywhere currently.